python/ray/tune/automlboard/backend/collector.py

```diff
@@ -70,15 +60,8 @@ def stop(self):
     @classmethod
     def init_logger(cls, log_level):
         """Initialize logger settings."""
-        logger = logging.getLogger("auomlboard")
-        logger.setLevel(log_level)
-        logging.getLogger().setLevel(logging.getLevelName(log_level))
-        logging.getLogger('requests.packages.urllib3.connectionpool') \
-            .setLevel(logging.WARNING)
-
-        logging.getLogger().handlers = []
-        log_format = '[%(asctime)s %(levelname)s] %(filename)s:' \
-                     '%(funcName)s:%(lineno)d  %(message)s'
+        log_format = '[%(levelname)s %(asctime)s] %(filename)s:' \
+                     '%(lineno)d  %(message)s'
         logging.basicConfig(
             stream=sys.stdout, format=log_format, level=log_level)
         return log_level
```

Comment:
should we return a logger here? and then use the Logger object in the rest of the class?